_posts/2024-02-09-release.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Release 2024.06
+author: Priya
+date: 2024-02-09
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Relocate the Delete button for views within the preview panel:
```

Comment:
View deletion is now available from within the preview panel

Reply:
But it was already there within the preview panel,we have changed only position @prmammoth .

Reply:
ok, got it